vnext/ReactUWP/Views/Image/ReactImage.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "ReactImage.h"
+
+namespace winrt {
+  using namespace Windows::UI;
+  using namespace Windows::UI::Xaml;
+  using namespace Windows::UI::Xaml::Controls;
+  using namespace Windows::UI::Xaml::Media;
+  using namespace Windows::Foundation;
+}
+
+namespace react {
+  namespace uwp {
+
+    ReactImage::ReactImage()
+    {
+      m_brush = ReactImageBrush::Create();
+      this->Background(m_brush.as<winrt::XamlCompositionBrushBase>());
```

Comment:
>XamlCompositionBrushBase [](start = 41, length = 24)

curious if we know the cost of using this compared to what Image was doing for us internally?  I thought Rob Meyer tried doing something like this to draw borders and it was expensive.. but maybe because we had so many borders.

Reply:
Using an XCBB + LoadedImageSurface will give us nearly identical performance.  XCBB does carry a little bit of extra overhead but I don't think we need to worry about this cost.  The more important cost is the underlying hardware texture... LoadedImageSurface internally uses the same downloading/caching code as Image so this should perform the same as Image.